standstill_action.py

```diff
@@ -0,0 +1,5 @@
+from talon import Module
+
+mod = Module()
+
+mod.list("clickless_mouse_action", "The list of actions that can be performed")
```

Comment:
Since this is specific to the single stage clicker, I might be inclined to move this definition into its python file.

And maybe rename to like clickless_mouse_single_stage_actions? Maybe not heh, that's very verbose.